### ISSUE TITLE
Input Device Only

### ISSUE THREAD

**Issue by ofeklivny** [AlvaroRojas/SkypeAudio#3]
Hi there!
I absolutely love the program and have used it everyday for almost a year.
Only frustrating problem I have with it is that it also changes Output Device without any way to disable it.
Meaning that while I change my input device from one microphone to another and it works great,
I also have to change device output back manually to my preferred output device, which is not the windows default one, every time I boot the computer.

I use "solicall" which is a great noise filter for incoming calls, so every time I boot It changes back to system default: my regular speakers and not through solicall filter which is annoying to change manually every day.

I have tried different versions with no sucess.

In short, I want to be able to choose: automatic output device switching: On\Off.
Or have a version without output changer, only Input changer.(and maybe the other way around too so other people with other preferences enjoy too.)

Thank you so much for providing us with this awesome tool. Appreciate you!

**Comment by AlvaroRojas** [comment]
It's a cool idea to be able to pick and choose what you want SkypeAudio to automatically change based on what the user selects.
I might develop soon™ a new version with those options without persistence, meaning it will not remember user selected options when closed.

I lack the time to develop a better solution right now, maybe in the future those options will persist bewteen application restarts.

**Comment by ofeklivny** [comment]
Thanks alot for you response! I'm currently trying to learn how to manually download the source code, remove any mention of "output device" and recompile it for my own use.
Will update here if it works, I'm not a developer/programmer so this is just me tinkering around because I really like the program.
